File: CODIGOS/codeabbey140.scala
fun countOnes(n: Int): Int {
  var count = 0
  for (i in 1..n) {
    for (j in 1..n) {
      if (i == j) {
        count++
      }
    }
  }
  return count
}

fun main(args: Array<String>) {
  val n = readLine()!!.toInt()
  println(countOnes(n))
}

File: CODIGOS/codeabbey140.pl
func countOnes(n: Int) -> Int {
  var count = 0
  for i in 1...n {
    for j in 1...n {
      if i == j {
        count += 1
      }
    }
  }
  return count
}

func main() {
  let n = Int(readLine()!)!
  print(countOnes(n))
}

main()
